morphct/system.py

```diff
@@ -96,6 +97,10 @@ def chromophores(self):
         """Return the chromophores in the system."""
         return self._chromophores
 
+    def carrier_data(self):
```

Comment:
```suggestion
    @property
    def carrier_data(self):
```
let's add the property tag